docker/manage

```diff
@@ -330,6 +340,8 @@ function setNgrokEndpoints() {
     NGROK_AGENT_PORT=4059
   fi
 
+  sleep 60
```

Comment:
We should not have arbitrary sleep times in the script. What is this supposed to do?

Reply:
It's the only thing allowing the build env to start properly.
I was thinking it's temporary. That was the new ticket I referred to in RC.

Reply:
Good eye though @esune

Reply:
If you want I can just comment it out for the PR... Then add a note beside it.

Reply:
I think we should figure out why this is needed, it should not be required to execute the script. DO you have any hints/ideas on why it is required? What happens when you remove it?